modules/core/src/lib/init.js

```diff
@@ -49,6 +49,3 @@ if (!global.deck) {
 if (!console.table) {
   console.table = () => {};
 }
-
-// Make sure we register shader modules
```

Comment:
Why did we have this? Was this needed to make sure we can refer to shadermodules by name?

Reply:
We already have `import './shaderlib';` inside the root index.js. I think this is just legacy code we forgot to remove.